pkg/kubernetes/persistentvolumeclaim/v1alpha1/persistentvolumeclaim.go

```diff
@@ -73,71 +73,78 @@ func (b *listBuilder) List() *pvcList {
 
 // Len returns the number of items present
 // in the PVCList
-func (p *pvcList) Len() int {
+func (p *PVCList) Len() int {
```

Comment:
In addition to this we can expose Len as a method from *ListBuilder.
Len will call List internally.
Refer to my earlier comments.

Reply:
Done added